packages/table/package.json

```diff
@@ -20,7 +20,7 @@
     "watch": "onchange 'src/**' 'preview/*.ts*' -- npm-run-all build:gulp build:preview"
   },
   "dependencies": {
-    "@blueprintjs/core": "^1.25.0",
+    "@blueprintjs/core": "^1.30.0",
```

Comment:
Bumping because of the utils that moved from `table` to `core`.

Reply:
is this the correct minimum version?

Reply:
Yup, as Chris mentioned, some utils moved from `table` to `core`

Reply:
Yes

Reply:
well sure but didn't that actually happen _last_ release? 1.29.0?

no big deal though.

Reply:
nvm, I see release notes. my bad.